cerberus/validator.py

```diff
@@ -97,6 +97,10 @@ class and a dictionary, the latter is passed to the
     """ Rules that will be processed in that order before any other and abort
         validation of a document's field if return ``True``.
         Type: :class:`tuple` """
+    post_priority_validation = ('dependencies',)
```

Comment:
though i introduced them, i'm not particular fond of these attributes, so i'd prefer not to introduce another one. i see that it has a small footprint, nonetheless i'd like to ask whether another approach is thinkable.